src/main/resources/META-INF/HotelTriples.txt

```diff
@@ -0,0 +1,60 @@
+name:JW Marriott	instanceOf	name:Marriott International, Inc.
```

Comment:
for now having this in resources is OK, but once we get your repro in the test side, I think we can reduce the size of this file as well once we start understanding the issue you encountered.